shared/macros.jinja

```diff
@@ -106,7 +106,11 @@ oc get {{% if all_namespaces %}}--all-namespaces{{% elif namespace %}}-n {{{ nam
 #}}
 {{% macro ocil_sshd_option(default, option, value) -%}}
     To determine how the SSH daemon's <tt>{{{ option }}}</tt> option is set, run the following command:
+    {{% if product in ['fedora', 'rhel9'] %}}
```

Comment:
@yuumasato shouldn't condition instead of checking prodtypes be `{{%- if sshd_distributed_config == "true" %}}`

Reply:
Fixed in c8e17a56ac66df0e143764037f78a4296c278345